datajoint/admin.py

```diff
@@ -21,11 +21,13 @@ def set_password(new_password=None, connection=None, update_config=None):   # pr
         config.save_local(verbose=True)
 
 
-def kill(restriction=None, connection=None):  # pragma: no cover
+def kill(restriction=None, connection=None, safemode=None):  # pragma: no cover
     """
     view and kill database connections.
     :param restriction: restriction to be applied to processlist
     :param connection: a datajoint.Connection object. Default calls datajoint.conn()
+    :param safemode: use interactive menu or programmatically terminate processes
```

Comment:
a better name would be `interactive` than `safemode`

Reply:
agree - though this is consistent with 'delete'... not sure what's best

Reply:
`delete` does not have a `safemode` argument. It has `verbose`. 

I think this can be implemented more intuitively as a separate method `kill_quick` or something.

Reply:
ah yes - got the argument confused with the configuration - either way, we use a term 'safemode' to denote 'prompting' elswhere in the codebase.

separating to 'kill quick' would also 'do' & be consistent elsewhere - will adjust.

Reply:
updated.